vulkan/vulkan_backbuffer.cc

```diff
@@ -52,14 +49,14 @@ bool VulkanBackbuffer::CreateSemaphores() {
   };
 
   auto semaphore_collect = [this](VkSemaphore semaphore) {
```

Comment:
If you are OK with the convention followed (that I described in an earlier comment). I can document that somewhere.